diff --git a/views/index.tt b/views/index.tt
index 9126935..9efe03c 100644
--- a/views/index.tt
+++ b/views/index.tt
@@ -45,7 +45,7 @@
               <div class="row margin-top2">
                 <div class="col-md-12">
                   <p class="h3"><span class="fa fa-check fa-1x"></span> <span class="statalign">[% stats.publications %]</span> <a href="/publication">[% h.loc("home.stats.publications") %]</a></p>
-                  <p class="h3"><span class="fa fa-check fa-1x"></span> <span class="statalign">[% stats.researchdata %]</span> <a href="/data">[% h.loc("home.stats.data_publications") %]</a></p>
+                  <!-- <p class="h3"><span class="fa fa-check fa-1x"></span> <span class="statalign">[% stats.researchdata %]</span> <a href="/data">[% h.loc("home.stats.data_publications") %]</a></p> -->
                   <p class="h3"><span class="fa fa-check fa-1x"></span> <span class="statalign">[% stats.oahits %]</span> <a href="/publication?q=fulltext=1">[% h.loc("home.stats.oa_publications") %]</a></p>
                   <p class="h3"><span class="fa fa-check fa-1x"></span> <span class="statalign">[% stats.projects %]</span> <a href="/project">[% h.loc("home.stats.projects") %]</a></p>
                 </div>
