diff --git a/views/footer.tt b/views/footer.tt
index c9d06bd..9ae46dc 100644
--- a/views/footer.tt
+++ b/views/footer.tt
@@ -5,17 +5,21 @@
 </main>
 
 <footer class="panel-footer">
-  <div class="container">
-    <div class="row">
-      <div class="col-sm-12">
-        <div class="col-md-4 col-sm-5">
-          [% FOR feature IN h.config.features.views.footer %]
-          [% tmpl =  feature.template _ '.tt'; INCLUDE $tmpl %]
-          [% END %]
+    <div class="container">
+        <div class="row">
+            <div class="col-sm-12">
+                <div class="col-md-4" style="text-align: center">
+                    <p><a href="https://www.uni-goettingen.de"><img src="/images/logo_uni_goettingen.png" alt=""></a></p>
+                </div>
+                <div class="col-md-4" style="text-align: center">
+                    <p><a href="https://www.sub.uni-goettingen.de"><img src="/images/logo_sub_goettingen.png" alt=""></a></p>
+                </div>
+                <div class="col-md-4" style="text-align: center">
+                    <p>[% h.loc("footer.powered_by") %]&nbsp;&nbsp;<a href="//librecat.org/"><img src="/images/librecat_logo.png" alt="" height="50px"></a></p>
+                </div>
+            </div>
         </div>
-      </div>
     </div>
-  </div>
 </footer>
 <div class="container" style="margin-top: 1em">
     <div class="row">
