diff --git a/scss/main.scss b/scss/main.scss
index 0ed9c0d..b6384dd 100644
--- a/scss/main.scss
+++ b/scss/main.scss
@@ -3,7 +3,7 @@ $icon-font-path: '/fonts/';
 /* @import "node_modules/bootstrap-sass/assets/stylesheets/bootstrap"; */
 // @import "bielestrap";
 $brand-primary: #163776;
-
+$font-family-sans-serif: 'Roboto', sans-serif;
 @import "bootstrap-sass/_bootstrap.scss";
 @import "bootstrap-theme";
 @import "bootstrap-select";
diff --git a/views/head_meta.tt b/views/head_meta.tt
index 0416464..aa3a62e 100644
--- a/views/head_meta.tt
+++ b/views/head_meta.tt
@@ -9,7 +9,7 @@
     <![endif]-->
     <link rel="stylesheet" href="/css/main.css">
     <!-- <link rel="stylesheet" href="/css/bootstrap-select.min.css"> -->
-    <link href="https://fonts.googleapis.com/css?family=Open+Sans:300,300i,400,400i,600,600i,700,700i" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css?family=Roboto:400,400i,700,700i&amp;subset=latin-ext" rel="stylesheet">
     <script src="/javascripts/jquery.min.js"></script>
     <script src="/javascripts/jquery-ui-1.10.4.custom.min.js"></script>
     <script src="/javascripts/bootstrap.min.js"></script><!-- main bootstrap file -->
