diff --git a/views/footer.tt b/views/footer.tt
index 6abb8a4..646d6d6 100644
--- a/views/footer.tt
+++ b/views/footer.tt
@@ -23,6 +23,13 @@
     </div>
   </div>
 </footer>
+<div class="container" style="margin-top: 1em">
+    <div class="row">
+        <div class="col-sm-12 small text-uppercase">
+            <p>Except where otherwise noted, content on this site is licensed under a <a href="https://creativecommons.org/licenses/by/4.0/">Creative Commons Attribution 4.0 International</a> license.</p>
+        </div>
+    </div>
+</div>
 </body>
 
 <!-- END footer.tt -->
