diff --git a/views/topbar.tt b/views/topbar.tt
index 3a9bb36..308f32f 100644
--- a/views/topbar.tt
+++ b/views/topbar.tt
@@ -50,7 +50,7 @@
                 [%- IF session.user %]
                 <li><a href="/logout"><span class="glyphicon glyphicon-log-out"></span><span class="hidden-sm">[% h.loc("main_page.logout") %]</span></a></li>
                 [%- ELSE %]
-                <li><a href="/login"><span class="glyphicon glyphicon-log-out"></span><span class="hidden-sm">[% h.loc("main_page.login") %]</span></a></li>
+                <li><a href="/login"><span class="glyphicon glyphicon-log-in"></span><span class="hidden-sm">[% h.loc("main_page.login") %]</span></a></li>
                 [%- END %]
             </ul>
         </div><!-- /.navbar-collapse -->
