diff --git a/views/hits.tt b/views/hits.tt
index b4a567f0..c6e341f4 100644
--- a/views/hits.tt
+++ b/views/hits.tt
@@ -82,7 +82,7 @@
     <em>[% IF entry.year %][% entry.year %][% ELSE %][n.d.][% END %] | [% lf.forms.item(entry.type).label %] | [% h.loc("hits.id") %]: <span class="pubid">[% entry._id %]</span> </em>
     <div class="citation-block-div">
     [%- IF style == 'short' %]
-	  <a href="[% h.host %]/[% IF backend %]librecat/record/edit[% ELSE %][% bag %][% END %]/[% entry._id %]"[% IF request.path.match("/embed") %] target="_blank"[% END %]><strong>[% entry.title %]</strong></a><br>
+	  <a href="/[% IF backend %]librecat/record/edit[% ELSE %][% bag %][% END %]/[% entry._id %]"[% IF request.path.match("/embed") %] target="_blank"[% END %]><strong>[% entry.title %]</strong></a><br>
     [%- END %]
     [%- IF backend AND ((!entry.locked AND !entry.type.match("^bi")) OR session.role == "super_admin") %]
       <a href="/librecat/record/edit/[% entry._id %]" onclick="window.top.location='/librecat/record/edit/[% entry._id %]'" class="citation-block-link"></a>
