From a873e3be0c32d1072156deff47572ab44df7d64f Mon Sep 17 00:00:00 2001
From: Vitali Peil <vitali.peil@uni-bielefeld.de>
Date: Tue, 21 Feb 2017 15:54:49 +0100
Subject: [PATCH] stupid (mistake)

---
 lib/LibreCat/Citation.pm | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/lib/LibreCat/Citation.pm b/lib/LibreCat/Citation.pm
index 1eccf6b..bf69735 100644
--- a/lib/LibreCat/Citation.pm
+++ b/lib/LibreCat/Citation.pm
@@ -59,7 +59,7 @@ sub _build_conf {
 sub _build_styles {
     my ($self) = @_;
     if ($self->all) {
-        return keys %{$self->conf->{csl}->{styles}};
+        return [keys %{$self->conf->{csl}->{styles}}];
     }
     elsif ($self->style) {
         return [$self->style];
