From 115460a64d435d0177d71c27ce656ac08c694f0f Mon Sep 17 00:00:00 2001
From: Vitali Peil <vitali.peil@uni-bielefeld.de>
Date: Fri, 10 Mar 2017 14:28:25 +0100
Subject: [PATCH] fix count for department publications, GH #195

---
 lib/LibreCat/Cmd/generate.pm | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/lib/LibreCat/Cmd/generate.pm b/lib/LibreCat/Cmd/generate.pm
index 3c9540a..d8929d9 100644
--- a/lib/LibreCat/Cmd/generate.pm
+++ b/lib/LibreCat/Cmd/generate.pm
@@ -190,7 +190,7 @@ sub _generate_departments {
             }
 
             my $id    = $item->{_id};
-            my $hits  = $pubs->search(query => "department:$id" );
+            my $hits  = $pubs->search(cql_query => "department=$id" );
             my $total = $hits->{total};
 
             $root->{tree}->{$id}->{name}    = $item->{name};
