diff --git a/lib/LibreCat/App/Search/Route/person.pm b/lib/LibreCat/App/Search/Route/person.pm
index 695d9d3..c084b7a 100644
--- a/lib/LibreCat/App/Search/Route/person.pm
+++ b/lib/LibreCat/App/Search/Route/person.pm
@@ -16,7 +16,7 @@ use URI::Escape;
 List persons alphabetically
 
 =cut
-get qr{/person/*([a-z,A-Z])*} => sub {
+get qr{/person/([a-z,A-Z]{1,1})$} => sub {
     my ($c) = splat;
 
     my $cql = $c ? ["lastname=" . lc $c . "*"] : '';
@@ -53,7 +53,7 @@ research data and author IDs.
 
 =cut
 
-get qr{/person/(\d+|\w+|[a-fA-F\d]{8}(?:-[a-fA-F\d]{4}){3}-[a-fA-F\d]{12})/*(\w+)*/*}
+get qr{/person/(.*?)/*(\w+)*/*}
     => sub {
     my ($id, $modus) = splat;
     my $p      = h->extract_params();
